src/DetailsView/components/adhoc-tab-stops-test-view.tsx

```diff
@@ -66,6 +76,7 @@ export const AdhocTabStopsTestView = NamedFC<AdhocTabStopsTestViewProps>(
                 </h1>
                 {description}
                 <RequirementInstructions howToTest={howToTest} />
+                <TabStopsFailedInstanceSection {...props} />
```

Comment:
nit: we usually do not use the spread operator for props. I think we usually only do that for components that kind of act like a switch or a wrapper.